TimberWinR/Outputs/Redis.cs

```diff
@@ -183,6 +201,18 @@ private bool ApplyFilters(JObject json)
             return drop;
         }
 
+
+        // 
+        // What is average queue depth?
+        //
+        private int AverageQueueDepth()
+        {
+            lock(_locker)
+            {
+                return (int)_sampleQueueDepths.Average();
```

Comment:
Not really, since it cannot ever get bigger than maxDepthAnyhow

On Tue, Mar 3, 2015 at 10:52 AM, Chris Baldauf <notifications@github.com>
wrote:

> In TimberWinR/Outputs/Redis.cs
> <https://github.com/Cimpress-MCP/TimberWinR/pull/31#discussion_r25698060>:
>
> > @@ -183,6 +201,18 @@ private bool ApplyFilters(JObject json)
> >              return drop;
> >          }
> >
> > +
> > +        //
> > +        // What is average queue depth?
> > +        //
> > +        private int AverageQueueDepth()
> > +        {
> > +            lock(_locker)
> > +            {
> > +                return (int)_sampleQueueDepths.Average();
>
> Forgive my C# lack of knowledge, but is there a risk of overflow since the
> _sampleQueueDepths is a long array and the the return value is an int?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cimpress-MCP/TimberWinR/pull/31/files#r25698060>.
>